extras/wait-for-bricks.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+#Copyright (c) 2018 Red Hat, Inc. <http://www.redhat.com>
+#This file is part of gluster-block.
+#
+#This file is licensed to you under your choice of the GNU Lesser
+#General Public License, version 3 or any later version (LGPLv3 or
+#later), or the GNU General Public License, version 2 (GPLv2), in all
+#cases as published by the Free Software Foundation.
+
+# This file waits for gluster block hosting volume bricks to be up. If all the
+# bricks are up, it exits with success status, otherwise waits for the $timeout
+# seconds before erroring out with the status about
+# online-bricks/expected-online-bricks
+
+LOGDIR="${GB_LOGDIR:-/var/log/gluster-block}"
+
+function printLog()
+{
+    local msg=${1}
+
+    echo "[$(date -u +'%Y-%m-%d %I:%M:%S')] ${msg}" >> "${LOGDIR}/gluster-block-bricks-start.log"
+}
+function volinfo_field()
+{
+    local vol=$1;
+    local field=$2;
+
+    gluster volume info "$vol" --xml | grep -oPm1 "(?<=<$field>)[^<]+"
+}
+
+function is_enabled()
+{
+        case $1 in
```

Comment:
Variables used in expressions should be placed between quotes (like `"$1"`) to prevent problems. In case `$1` is an empty string, the script will error out.